tests/README.md

```diff
@@ -35,15 +35,15 @@ sudo gpasswd -a runner rkt
 #### Thread 1
 
 ```
-./tests/run-build.sh none
-./tests/run-build.sh src v229
+./tests/run-build.sh -f none -c
+./tests/run-build.sh -f src -s v229 -c
```

Comment:
Please describe somewhere there in docs what each flag means (maybe simple copy+paste source/usage).